### ISSUE TITLE
The plugin doesn't send all the events

### ISSUE THREAD

**Issue by jkulak** [logtail/fluentd-plugin-logtail#3]
Hi, I have described the issue and prepared testing code here: https://github.com/jkulak/logtail-fluentd-plugin-test

My `fluentd` configuration is properly sending logs to a file and it's `stdout`, but only some of the events are being sent to Logtail.

**Comment by ivan-section-io** [comment]
Yes. For me it also does "not work" at the moment.

It appears that it POSTs concatenated msgpacks as "Content-Type: application/msgpack", but only the first is being decoded at the receiving end.

The example of multiple events at https://docs.logtail.com/integrations/rest-api#example-multiple-events is not concatenated msgpacks of objects - ie `msgpack({"message": "A"})msgpack({"message": "B"})` (as the plugin does), but a msgpack of an array of objects - ie`msgpack([{"message": "A"},{"message": "B"}])`.

The result is that only the first message in any batch/chunk appears in logtail.

Either the server end needs to *ALSO* decode any remaining bytes - or the plugin needs to change its approach.

**Comment by ivan-section-io** [comment]
By determining the number of records, and adding a suitable msgpack array header - all records appear in logtail.
```diff
--- /lib/fluent/plugin/out_logtail.rbout_logtail.rb
+++ /lib/fluent/plugin/out_logtail.rbout_logtail.rb
@@ -38,7 +38,13 @@
         end

         http = build_http_client
-        body = chunk.read
+
+        records=0
+        chunk.each do
+          records=records+1
+        end
+        body = [0xdd,records].pack("CN")
+        body << chunk.read

         begin
           resp = http.start do |conn|
```
Deserialising objects just to count them feels inefficient. You could also deserialise them, build an array and msgpack serialise that - also feels like double handling.
I couldn't find where chunks intentionally expose the number of records - but didn't look too hard (I don't know ruby).

**Comment by adikus** [comment]
Thanks a lot @ivan-section-io. I've released a new version with your fix as 0.2.0. cc @jkulak sorry about taking so long to get to this.
